scipy/stats/tests/test_discrete_basic.py

```diff
@@ -88,10 +88,11 @@ def test_moments(distname, arg):
 
     # compare `stats` and `moment` methods
     check_moment(distfn, arg, m, v, distname)
-    check_mean_expect(distfn, arg, m, distname)
-    check_var_expect(distfn, arg, m, v, distname)
-    check_skew_expect(distfn, arg, m, v, s, distname)
-    if distname not in ['zipf', 'yulesimon']:
+    if distname not in ['betanbinom']:
+        check_mean_expect(distfn, arg, m, distname)
+        check_var_expect(distfn, arg, m, v, distname)
+        check_skew_expect(distfn, arg, m, v, s, distname)
+    if distname not in ['zipf', 'yulesimon', 'betanbinom']:
```

Comment:
see above, this is not needed if we change the default parameters. I tried to run the test with 5, 9.3, 1 and it was ok

Reply:
For me, with $a>8$, all tests pass except the kurtosis with (5, 9.3, 1). Also with other parameters I cannot get the kurtosis test to pass.

Reply:
strange, I did not run the test itself, I just copied the functionality from `common_tests.py` which should check the following:

```
dist = betanbinom(5, 9.3, 1)
m, v, s, k = dist.stats('mvsk')

if np.isfinite(k):
    m4e = dist.expect(lambda x: np.power(x-m, 4))
    assert_allclose(m4e, (k + 3.) * np.power(v, 2),
                        atol=1e-5, rtol=1e-5,
                        err_msg='kurtosis')
elif not np.isposinf(k):
    assert np.isnan(k)
```

and check does not fail when I run it

Anyways, we have enough checks around kurtosis now